students/luftsmeerflier/lesson04/mailroom_2.py

```diff
@@ -0,0 +1,134 @@
+#/usr/bin/env python3
+import sys
+
+donors = {
+	"John Quigley": [
+		("2019-02-04", 5000.00),
+		("2019-02-04", 4250.00),
+		("2019-02-04", 3000.00)
```

Comment:
You can simplify this structure and use a dictionary:
`{"Bob": {"2019-02-04": 5000, "2019-02-04": 4250}}`

then you can get all donations by values `donors.values()`and make your code much simpler